perun-core/src/main/java/cz/metacentrum/perun/core/impl/Utils.java

```diff
@@ -1117,11 +1117,11 @@ public static void sendSMS(String telNumber, String message) throws InternalErro
 			// handle response
 			if (exitValue == 0) {
 				// successful
-				log.info("SMS with text \"" + message + "\" to tel. number " + telNumber + " successfully sent.");
+				log.debug("SMS with text \"" + message + "\" to tel. number " + telNumber + " successfully sent.");
```

Comment:
Please pass ``message`` and ``telNumber`` like params into debug method and not as joined string with ``+``.